packages/page-template-modal/src/utils/test/group-utils.js

```diff
@@ -0,0 +1,41 @@
+/**
```

Comment:
thank you for adding this test!

Reply:
np :) I wanted to remove most of our lodash uses, but wasn't confident to do the refactor without writing the tests first :D